### DISCUSSION TITLE
Add skipFetchSetup parameter to (at least) jsonRpcProvider

### DISCUSSION THREAD

**Discussion by vorcigernix** [wevm/wagmi#717] [Ideas]
Hi, for the reasons stated here: https://github.com/ethers-io/ethers.js/issues/1886 I'd like to use this parameter with providers. It was patched to ethers while ago. Change seems to be trivial, I was going to create PR but the contributing guide states that I should begin with discussion. 
Change looks like adding two lines of code to the file https://github.com/tmm/wagmi/blob/main/packages/core/src/providers/jsonRpc.ts but I haven't tested locally. If you agree to turn this to PR, I'll do it.

**Comment by tmm** [comment]
Sounds good!
